### PR TITLE
Configure subtitle cache after "erase subtitle cache" action

### PR DESCRIPTION
**Pull request by fernandog**
- [ ] PR is based on the DEVELOP branch
- [ ] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [ ] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: medusa/__main__.py [hunk 1]
```diff
@@ -1094,6 +1090,10 @@ def initialize(self, console_logging=True):
                 # Disable flag to erase cache
                 app.SUBTITLES_ERASE_CACHE = 0
```
Comment (sharkykh) [inline medusa/__main__.py]: Should be `app.SUBTITLES_ERASE_CACHE = False`

File: medusa/__main__.py [hunk 2]
```diff
@@ -1094,6 +1090,10 @@ def initialize(self, console_logging=True):
...
 
+            # Check if we need to perform a restore of the cache folder
+            Application.restore_cache_folder(app.CACHE_DIR)
```
Comment (sharkykh) [inline medusa/__main__.py]: Keep these two lines where they we're. This is the restore feature. You only need the next line here IMO.

**Comment by sharkykh** [comment]
Fixes #2449, #4008, #4009

File: medusa/__main__.py (hunk 2 above)
Comment (fernandog) [reply medusa/__main__.py]: If we restore before we could erase if the erase setting is enabled. Right?

Comment (sharkykh) [reply medusa/__main__.py]: Yes. But, I mean.. in that case, what do you think the user wants to do?Restore the old cache or remove it?

Comment (sharkykh) [reply medusa/__main__.py]: Oh, I see what you mean now...
I guess you're right :+1:

**Pull request merged**